data/json/items/resources/misc.json

```diff
@@ -250,8 +250,8 @@
     "material": [ "flesh" ],
     "volume": "250 ml",
     "cutting": 8,
-    "flags": [ "SPEAR", "SHEATH_KNIFE" ],
-    "weapon_category": [ "SHIVS" ],
+    "flags": [ "SPEAR", "SHEATH_KNIFE", "ALLOWS_BODY_BLOCK" ],
+    "weapon_category": [ "SHIVS", "ALLOWS_BODY_BLOCK" ],
```

Comment:
"weapon_category" probably shouldn't have "ALLOWS_BODY_BLOCK" flag)

Reply:
Thank you! I had not realized I typed the flag where it didn't belong...